doc/make_doc.in

```diff
@@ -16,7 +16,7 @@ echo "--------------------"
 
 "$GAP" $GAPARGS -A <<EOF
 base:="@abs_top_srcdir@";;
-books:=["ref", "tut", "changes", "hpc", "dev"];;
```

Comment:
There is at least two more files that need to be adapted: `lib/helpbase.gi` and `hpcgap/lib/helpbase.gi`

Reply:
I've found a couple more using `git grep "tut.*changes" and added a corresponding commit to this PR.